src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,102 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int GROWTH_COEFFICIENT = 2;
+    private int capacity = 16;
+    private int threshold = (int) (capacity * LOAD_FACTOR);
+    private int size;
+    private Node<K, V>[] table = new Node[capacity];
+
+    private static class Node<K, T> {
+        private K key;
+        private T value;
+        private Node<K, T> next;
+
+        public Node(K key, T value, Node<K, T> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
+
+    private int findIndex(K key) {
```

Comment:
```suggestion
    private int getIndex(K key) {
```

private methods should follow after public as helper ones

Reply:
isn't it bad to call methods like getSomething? To me personally they start to look like getters, although they are not.

Reply:
Не зустрічала таких правил, що не можна використовувати слово get, але назва методу має відображати що саме робить метод. В даному випадку він нічого не шукає, а рахує індекс можеш назвати якось computeIndex або щось таке